curriculum/challenges/english/20-upcoming-python/learn-list-comprehension-by-building-a-case-converter-program/657f4a4a5828a01de04b652f.md

```diff
@@ -0,0 +1,52 @@
+---
+id: 657f4a4a5828a01de04b652f
+title: Step 21
+challengeType: 20
+dashedName: step-21
+---
+
+# --description--
+
+Get rid of the commented lines of code inside the `convert_to_snake_case()` function to clean up the function definition.
+
+# --hints--
+
+You should remove the commented out lines of code.
+
+```js
+
```

Comment:
Just FYI, I would do something like this

```suggestion
const transformedCode = e.original.code["main.py"].replace(/\r/g, '');
const sansComments = __helpers.python.removeComments(transformedCode);
assert.notMatch(sansComments, /snake_cased_char_list.append\(/);
```

Reply:
Thanks for the tip Shaun.